### ISSUE TITLE
[CI] `bulk/11_basic_with_types/Empty _id with op_type create` failure

### ISSUE THREAD

**Issue by romseygeek** [elastic/elasticsearch#48396]
```
REPRODUCE WITH: ./gradlew ':x-pack:qa:core-rest-tests-with-security:integTestRunner' --tests "org.elasticsearch.xpack.security.CoreWithSecurityClientYamlTestSuiteIT.test {yaml=bulk/11_basic_with_types/Empty _id with op_type create}" \
  -Dtests.seed=BAD1058B2354254D \
  -Dtests.security.manager=true \
  -Dtests.locale=ko-KR \
  -Dtests.timezone=Navajo \
  -Dcompiler.java=12 \
  -Druntime.java=8 \
  -Dtests.rest.blacklist=index/10_with_id/Index with ID,indices.get_alias/10_basic/Get alias against closed indices
```

```
1>           "create" : {
1>             "_index" : "test",
1>             "_type" : "_doc",
1>             "_id" : "sy12-G0BLlkbJhm26u1C",
1>             "status" : 400,
1>             "error" : {
1>               "type" : "illegal_argument_exception",
1>               "reason" : "Rejecting mapping update to [test] as the final mapping would have more than 1 type: [type, _doc]"
1>             }
1>           }
1>         }
```

The bulk request sent at the start of this test includes a `type` for the first four entries, but not for the fifth.  I'm not sure why this works some of the time and not others, but it should be easily fixable by making all bulk entries consistent in respect of type information.

**Comment by ywelsch** [comment]
Build failure: https://gradle-enterprise.elastic.co/s/xwp4oczhdcvaw

**Comment by ywelsch** [comment]
Closed by #48418
